src/test/java/net/masterthought/cucumber/generators/TagsOverviewPageTest.java

```diff
@@ -66,13 +66,14 @@ public void prepareReport_setTagsToExcludeFromChart_ReturnsFilteredTags() {
 
         // then
         VelocityContext context = page.context;
-        assertThat(context.get("chart_categories")).isEqualTo("[\"@fast\"]");
-        assertThat(context.get("chart_data")).isEqualTo(asList(
-                "[100.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]"));
+        assertThat(context.get("chart_categories")).isEqualTo(new String[]{"@fast"});
```

Comment:
Can do a cast and then use containsExactly but that doesn't improve readability.  Example:

```java
        assertThat((String[]) context.get("buildNumbers")).containsExactly("01_first", "other build", "05last");
        assertThat((int[]) context.get("failedFeatures")).containsExactly(1, 2, 5);
        assertThat((int[]) context.get("totalFeatures")).containsExactly(10, 20, 30);

        assertThat((int[]) context.get("failedScenarios")).containsExactly(10, 20, 20);
        assertThat((int[]) context.get("totalScenarios")).containsExactly(10, 2, 5);

        assertThat((int[]) context.get("passedSteps")).containsExactly(1, 3, 5);
        assertThat((int[]) context.get("failedSteps")).containsExactly(10, 30, 50);
        assertThat((int[]) context.get("skippedSteps")).containsExactly(100, 300, 500);
        assertThat((int[]) context.get("pendingSteps")).containsExactly(1000, 3000, 5000);
        assertThat((int[]) context.get("undefinedSteps")).containsExactly(10000, 30000, 50000);

        assertThat((long[]) context.get("durations")).containsExactly(3206126182398L, 3206126182399L, 3206126182310L);
```